holidays/countries/india.py

```diff
@@ -28,269 +37,596 @@ class India(HolidayBase, ChristianHolidays, InternationalHolidays, IslamicHolida
     """
 
     country = "IN"
+    default_language = "en_IN"
+    supported_languages = ("en_IN", "en_US", "hi")
```

Comment:
```suggestion
    # %s (estimated).
    estimated_label = tr("%s (estimated)")
    supported_languages = ("en_IN", "en_US", "hi")
```
As the country has movable holidays.

Reply:
Thanks for the suggestion!
I was wondering if I should add this estimated label to all ranged holidays, including all national and subdivision-level ranged holidays. 
If yes, how do I implement it?

Reply:
You just define the `estimated_label` variable, and everything else is is automatically done. But display of this label is enabled by default only for IslamicHolidays.

Reply:
Yes, I slightly forgot about the categories. If you know exactly which holidays are restricted, you can move them to the `OPTIONAL` category. You can look at [Denmark](https://github.com/vacanza/holidays/blob/dev/holidays/countries/denmark.py) as an example of country with categories. (Also, remember to add `OPTIONAL` category to description of India in README)

Reply:
Okayy.. Well I have an exam tomorrow so I'll Implement it in upcoming days